app/components/map/index.js

```diff
@@ -135,31 +141,21 @@ class Map extends Component {
 
   onLayout = () => {
     if (!this.state.alertSelected) {
-      if (this.afterRenderTimer) {
-        clearTimeout(this.afterRenderTimer);
+      if (this.areaFeatures && this.areaFeatures.length > 0) {
+        const area = this.getAreaCoordinates(this.areaFeatures[this.state.index]);
+        this.map.fitToCoordinates(area,
+          { edgePadding: { top: 250, right: 250, bottom: 250, left: 250 }, animated: true });
       }
-      this.afterRenderTimer = setTimeout(() => {
-        if (this.areaFeatures && this.areaFeatures.length > 0) {
-          const area = this.getAreaCoordinates(this.areaFeatures[this.state.index]);
-
-          this.map.fitToCoordinates(area,
-            { edgePadding: { top: 250, right: 250, bottom: 250, left: 250 }, animated: true });
-        }
-      }, 300);
     }
   }
 
+  // TODO: check if we really need to map events
```

Comment:
We need this to update the coords of the map and be able to get the zoom level params to the tiles implementation, so yes, we really need this